src/garage/torch/algos/sac.py

```diff
@@ -205,92 +214,144 @@ def train_once(self, itr=None, paths=None):
         """
         del itr
         del paths
-        if self.replay_buffer.n_transitions_stored >= self.min_buffer_size:
-            samples = self.replay_buffer.sample(self.buffer_batch_size)
+        if self._buffer_prefilled:
+            samples = self.replay_buffer.sample_transitions(self.buffer_batch_size)
             samples = tu.dict_np_to_torch(samples)
             policy_loss, qf1_loss, qf2_loss = self.optimize_policy(0, samples)
             self._update_targets()
 
         return policy_loss, qf1_loss, qf2_loss
 
-    def _get_log_alpha(self, **kwargs):
+    def _get_log_alpha(self, samples_data):
         """Return the value of log_alpha.
 
+        Args:
+            samples_data (dict): Transitions(S,A,R,S') that are sampled from
+                the replay buffer. It should have the keys 'observation',
+                'action', 'reward', 'terminal', and 'next_observations'.
+
         This function exists in case there are versions of sac that need
         access to a modified log_alpha, such as multi_task sac.
 
-        Args:
-            kwargs(dict): keyword args that can be used in retrieving the
-                log_alpha parameter. Unused here.
+        Note:
+            samples_data's entries should be torch.Tensor's with the following
+            shapes:
+                observation:
+                    (sac.env_spec.observation_space, sac.buffer_batch_size)
```

Comment:
Please use the style for documenting shapes established in `src/garage/_dtypes.py`